enterprise/frontend/src/embedding-sdk/components/public/CreateDashboardModal/CreateDashboardModal.stories.tsx

```diff
@@ -24,11 +24,7 @@ export default {
 };
 
 const Template: StoryFn<ComponentProps<typeof CreateDashboardModal>> = args => (
-  <CreateDashboardModal
-    onClose={action("onClose")}
-    onCreate={action("onCreate")}
```

Comment:
These are already included with Storybook by default, we can still see `actions` even without doing this.
![image](https://github.com/user-attachments/assets/da4c1b9b-268a-4949-bd47-3d20d089e869)